pkg/codegen/docs/templates/properties.tmpl

```diff
@@ -1,48 +1,22 @@
-{{ define "propIsRequired" }}
-{{ if . }} (Required) {{- else }} (Optional){{- end }}
-{{ end }}
-
-{{ define "properties_table" }}
-<table class="ml-6">
-    <thead>
-        <tr>
-            <th>Argument</th>
-            <th>Type</th>
-            <th>Description</th>
-        </tr>
-    </thead>
-    <tbody>
-    {{ range . }}
-        <tr>
-            <td class="align-top">{{ htmlSafe .DisplayName }}</td>
-            <td class="align-top">
-                {{ if eq .Type.Link "#" "" }}
-                <code>{{ htmlSafe .Type.Name }}</code>
-                {{- else }}
-                <code>{{ template "linkify" .Type }}</code>
-                {{- end }}
-            </td>
-            <td class="align-top">
-            {{- htmlSafe "{{% md %}}" }} {{ if .IsInput }}{{ template "propIsRequired" .IsRequired }}{{- end }} {{- htmlSafe .Comment }} {{ htmlSafe "{{% /md %}}" }}
-
-            {{ .DeprecationMessage }}
-            </td>
-        </tr>
-    {{ end }}
-    </tbody>
-</table>
-{{ end }}
-
 {{ define "properties" }}
 
-{{ htmlSafe "{{< langchoose csharp nojavascript >}}" }}
-
 {{ range $lang, $props := . }}
-{{ print "{{% lang" }} {{ print $lang }} {{ print "%}}" }}
-
-{{ template "properties_table" $props }}
-
-{{ print "{{% /lang %}}" }}
+{{ print "{{% choosable language" }} {{ print $lang }} {{ print "%}}" }}
+<dl class="resources-properties">
+{{ range . }}
+    <dt class="property-{{- if .IsInput -}}{{- if .IsRequired -}}required{{- else -}}optional{{- end -}}{{- end -}}{{ if .DeprecationMessage }} property-deprecated{{- end -}}"
+            title="{{ if .IsInput -}}{{- if .IsRequired -}}Required{{- else -}}Optional{{- end -}}{{- end -}}{{- if .DeprecationMessage -}}, Deprecated{{- end -}}">
+        <span>{{- htmlSafe .DisplayName -}}</span>
+        <span class="property-indicator"></span>
+        <span class="property-type">{{- if eq .Type.Link "#" "" -}}{{- htmlSafe .Type.Name -}}{{- else -}}{{ template "linkify" .Type }}{{- end -}}</span>
+    </dt>
+    <dd>
+        {{- print "{{% md %}}" -}}{{- .Comment -}}{{- print "{{% /md %}}" -}}
```

Comment:
I think this should be:
```suggestion
        {{- print "{{% md %}}" -}}{{- htmlSafe .Comment -}}{{- print "{{% /md %}}" -}}
```

...this is needed to fix this issue: https://github.com/pulumi/pulumi/issues/4177